packages/storybook/src/nimble/table/table-matrix.stories.ts

```diff
@@ -158,6 +160,21 @@ export const tableMultipleSelectionThemeMatrix: StoryFn = createMatrixThemeStory
 );
 tableMultipleSelectionThemeMatrix.play = playFunction;
 
+const tableKeyboardFocusStates = cartesianProduct([
+    ['multiple'],
+    [groupedStates[0]],
+    [hierarchyStates[0]]
```

Comment:
Can you make named aliases for the states like was done in the button matrix so that it's easier to see in source what states are being picked, example:
https://github.com/ni/nimble/blob/d37ca77cda0729b42f05b4bac79be8ccc98ceb22/packages/storybook/src/nimble/button/button-matrix.stories.ts#L69-L74

Reply:
Done